staging/cse/windows/azurecnifunc.ps1

```diff
@@ -83,9 +83,11 @@ function Set-AzureCNIConfig
     } else {
         # Fill in DNS information for kubernetes.
         $exceptionAddresses = @()
-        if ($IsDualStackEnabled){
-            $subnetToPass = $KubeClusterCIDR -split ","
-            $exceptionAddresses += $subnetToPass[0]
+        if ($IsDualStackEnabled) {

```

Comment:
Please test this change with a private cse package and update the test result.

Reply:
ran a standalone test and verified:

1. the cluster cidr is passed in as comma sep list
  ```
  "Network": {
            "ClusterCidr": "10.240.0.0/16,fd81:6b3c:7bb7:5fbc::/64",
            "ServiceCidr": "10.0.0.0/16",
            "DnsIp": "10.0.0.10"
        }
  ```
2. the conflist contains the right exception list
```
"AdditionalArgs": [
                {
                    "Name": "EndpointPolicy",
                    "Value": {
                        "Type": "OutBoundNAT",
                        "ExceptionList": [
                            "10.240.0.0/16",
                            "fd81:6b3c:7bb7:5fbc::/64"
                        ]
                    }
                },
```